src/components/EditSections/EditContactInfo/EditContactInfo.js

```diff
@@ -84,16 +83,12 @@ const EditContactInfo = ({
         </Col>
         <Col xs={12} md={3}>
           <Field
-            label={(
-              <>
-                <FormattedMessage id="ui-users.contact.preferredContact" />
-                <Asterisk />
-              </>
-            )}
+            label={<FormattedMessage id="ui-users.contact.preferredContact" />}
```

Comment:
@BogdanDenis, I see that you just removed my changes. I did not put `required` attribute because those fields (`preferredContact`, `patronGroup`, `status`) are not `required` on BE side, so I guess it is not a good idea to make it `requred`. Please clarify this changes with BE team.

Reply:
Maybe you should try `aria-required="true"`, then screen reader will read this fields, but it can stay without `required` attribute. But I did not test it.

Reply:
@annamelnyk `preferredContact` and `patronGroup` are not marked as `required`, but in [`validate` function in `UserForm` component](https://github.com/folio-org/ui-users/blob/master/src/views/UserEdit/UserForm.js#L60) they are still being checked for empty value, so in theory the behavior hasn't changed.
If we need to make them nor required in UI, then we'll have to remove these checks in validation as well.
Anyways, I'll raise this with our BE team and PO (I think Khalilah is PO in your team also, right?)

Reply:
No, Khalilah is not our PO any more. 
But in my [ticket](https://issues.folio.org/browse/UIU-1381) was left the comment with suggestion not to make fields `required`, like they shoud stay "conditionally" required.
I guess the right way discuss it with BE team.